duckduckgo/duckduckgo-privacy-extension
# Remove legacy Click to Load configuration

The migration away from the legacy social_ctp_configuration.json
configuration for Click to Load is complete. Let's use the new
configuration for the MV2 code paths and remove all references to the
legacy configuration.

While at it, let's also:
 - Deprecate the clickToLoadClicks Object that's no longer required
   and remove the corresponding logic.
 - Remove the imgToData function that was added for Click to Load but
   no longer used.
 - Remove a now unused webNavigation.onCommitted listener that was
   originally needed for Click to Load.
 - Rewrite the Click to Load unit tests that were passing
   incorrectly.
 - Fix the Click to Load Facebook integration tests, so that they
   aren't marked as pending when blocking fails.




**Reviewer:** @sammacbeth 
Cc: @ladamski, @franfaccin 


## Steps to test this PR:
1. Ensure the Click to Load integration tests are still passing (for MV2 and MV3).
2. Test that Facebook Click to Load works correctly on the Facebook test page (for MV2).

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
